### PR TITLE
Patch 0.19.3

### LINKED ISSUE #922: [1] Create patch release for 0.19 and 0.20
Create patch release for 0.19 and 0.20 which includes fix for failing jobs.

The ones I know of is:
https://github.com/elastisys/compliantkubernetes-apps/pull/895
https://github.com/elastisys/compliantkubernetes-apps/pull/785
https://github.com/elastisys/compliantkubernetes-apps/pull/920
https://github.com/elastisys/compliantkubernetes-apps/pull/761
https://github.com/elastisys/compliantkubernetes-apps/pull/918

Some of these fixes might already be in a release so just add the ones that are needed to the respective release.

You might also do a quick check and see if there are any other fixes that can be included to reduce the amount of alerts. But the ones listed should be a good start

## Definition of done

* There are patch releases for 0.19 and 0.20 that reduces failed job alerts.

### PR DESCRIPTION
**Pull request by Pavan-Gunda**
**What this PR does / why we need it**:
Added new changes to version 0.19 
**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #922

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Add a screenshot or an example to illustrate the proposed solution:**

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [ ] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

**Comment by crssnd** [comment]
the [upgrade steps](https://github.com/elastisys/compliantkubernetes-apps/blob/patch-0.19.3/migration/v0.18.x-v0.19.x/upgrade-apps-0-19-3.md) still uses `v0.20.x-v0.21.x`

**Comment by crssnd** [comment]
I would also add the remove grafana-ops script in the main migration scripts, so it doesn't fail when we upgrade directly to 0.19.3

**Review by Xartos — APPROVED** [review]
LGTM

File: migration/v0.18.x-v0.19.x/upgrade-apps.md [hunk 1]
```diff
@@ -105,3 +105,11 @@
    ```bash
    bin/ck8s ops kubectl sc delete po influxdb-0 -n influxdb-prometheus
    ```
+
+1. Remove grafana-ops chart with all the dashboards: `migration/v0.18.x-v0.19.x/remove-grafana-ops.sh`
```
Comment (crssnd) [inline migration/v0.18.x-v0.19.x/upgrade-apps.md:109]: move this up above line 87, and remove lines 110-115

Comment (Pavan-Gunda) [reply migration/v0.18.x-v0.19.x/upgrade-apps.md]: Done!!

**Review by crssnd — APPROVED** [review]

**Pull request merged**